lib/app/config.rb

```diff
@@ -46,5 +46,25 @@
 module App
   DEFAULT_PASSWORD = 'changeme'
 
-  Config = ::Settings
+  # If it's on cloud (i.e. EnvVar ENT_SEARCH_CONFIG_PATH is set), elasticsearch config in ent-search will be used.
+  Config = ::Settings.tap do |config|
+    if ENV['ENT_SEARCH_CONFIG_PATH']
+      Utility::Logger.info("Found ENT_SEARCH_CONFIG_PATH, loading ent-search config from #{ENV['ENT_SEARCH_CONFIG_PATH']}")
```

Comment:
I purposely use `info` here because we want to know we overwrite elasticsearch config with ent-search config.

I think connectors-python [use `info`](https://github.com/elastic/connectors-python/blob/fa1f5208a6650d8196a348e5fff1d381b12c5b8f/connectors/runner.py#L52) as well